stories/design-guide.story.ts

```diff
@@ -0,0 +1,443 @@
+import { html, svg } from 'lit-html';
+import customProperties from './../custom-properties.json';
```

Comment:
Is this correct?
Its complaining about it, haven't cloned down, so that's why I'm just asking stupid question... but I was just giving a early look at this. :-)

"Unable to resolve path to module './../custom-properties.json'"

Reply:
It's complaining about something else I need to fix, I changed it do draft for now.

Reply:
This is fixed now